packages/grafana/build.sh

```diff
@@ -39,6 +39,12 @@ termux_step_post_get_source() {
 	go mod download github.com/hashicorp/go-sockaddr@$sockaddr_version
 	cp -r "$(go env GOPATH)"/pkg/mod/github.com/hashicorp/go-sockaddr@$sockaddr_version go-sockaddr
 	chmod +w -R go-sockaddr
+
```

Comment:
I think it is better to use `go mod vendor`.

Reply:
Thanks, this is much cleaner. Just pushed 177fc737124eca19b086e9b29f0fe7bb25f9c42f, do review it